### ISSUE TITLE
MPI3 Fault Tolerance - RMA

### ISSUE THREAD

**Issue by mpiforumbot** [mpi-forum/mpi-forum-historic#325]
**Originally by _jjhursey_ on 2012-03-06 20:27:55 -0600**

---
# #325-E: One-sided semantic after process failuresThis ticket is a split from ticket #323 . It covers the specific semantic of returned errors in case RMA operations cannot be completed after a process failure, and the recovery options for the associated window.
## Votes

no votes yet.
## Extended Scope

This is a split from ticket #323. It depends on ticket #323.
## History

Initially was part of ticket #323. It is now separated out. 
## Impact on Implementations

Adds the MPI_WIN_REVOKE, MPI_WIN_GET_FAILED functions and clarifies the memory semantics after a process failure.
## Impact on Applications / Users

Provides fault tolerance to interested users.
Users/implementations that do not care about fault tolerance are not impacted.

**Comment by mpiforumbot** [comment]
**Originally by _jjhursey_ on 2012-03-06 22:44:21 -0600**

---

**Attachment added:** [`mpi-report.pdf`](https://github.com/mpiforumbot/mpi-forum-historic/raw/master/attachments/325/mpi-report.pdf) (2322.3 KiB)
Changes for ticket 325, within the scope of ticket 323

**Comment by mpiforumbot** [comment]
**Originally by _bouteill_ on 2012-05-23 14:26:50 -0500**

---

This ticket is on hold. It is not going to be presented during the meeting in Japan.

**Comment by mpiforumbot** [comment]
**Originally by @wbland on 2013-10-24 10:29:52 -0500**

---

This ticket is no longer necessary as the entire FT proposal will be presented as #323 in the future rather than as separate tickets.

**Comment by mpiforumbot** [comment]
**Originally by @wbland on 2015-04-09 15:23:28 -0500**

---

It was decided to leave the RMA portion of ULFM as a separate ticket to prevent it from holding up the reading of the main ticket. Obviously, the intention is still to get all parts of FT into the standard at the same time, even if they aren't the same ticket (as has been done with other items in the past).
